guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -7,48 +7,145 @@ To view the exact set of permissions a predefined role grants, display the role
+endif::[]
+|{Project} automatically assigns this role to every user in the system.
+
+|Discovery Manager
+|View, provision, edit, and destroy discovered hosts.
+
+Manage discovery rules.
+|
+
+|Discovery Reader
+|View hosts and discovery rules.
+|
+
+|Edit hosts
+|View, create, edit, destroy, and build hosts.
+|
+
+|Edit partition tables
+|View, create, edit, and destroy partition tables.
+|
+
+|Manager
+|View and edit global settings.
+|
+
+|Organization admin
+|All permissions except permissions for managing organizations.
+|An administrator role defined per organization.
 The role has no visibility into resources in other organizations.
-| Red{nbsp}Hat Access Logs | View the log viewer and the logs.
-| Remote Execution Manager | Control which roles have permission to run infrastructure jobs.
-| Remote Execution User | Run remote execution jobs against hosts.
-| Site manager | A restrained version of the Manager role.
-| System admin a|
-* Edit global settings in *Administer* > *Settings*.
-* View, create, edit and destroy users, user groups, and roles.
-* View, create, edit, destroy, and assign organizations and locations but not view resources within them.
-
-Users with this role can create users and assign all roles to them.
+
+By cloning this role and assigning an organization, you can delegate administration of that organization to a user.
+
+|Red{nbsp}Hat Access Logs
+|View the log viewer and the logs.
+|
+
+|Remote Execution Manager
+|Control which roles have permission to run infrastructure jobs.
+|
+
+|Remote Execution User
+|Run remote execution jobs against hosts.
+|
+
+|Site manager
+|Mostly view permissions for various items.
+
+Permissions to manage hosts in the infrastructure.
+|A restrained version of the Manager role.
+
+|System admin
+|Edit global settings in *Administer* > *Settings*.
+
+View, create, edit, and destroy users, user groups, and roles.
+
+View, create, edit, destroy, and assign organizations and locations but not view resources within them.
+
+|Users with this role can create users and assign all roles to them.
 Therefore, ensure to give this role only to trusted users.
-| Tasks manager | View and edit {Project} tasks.
-| Tasks reader | A role that can only view {Project} tasks.
-| Viewer | A passive role that provides the ability to view the configuration of every element of the {Project} structure, logs, reports, and statistics.
-| View hosts | A role that can only view hosts.
-| Virt-who Manager | A role with full virt-who permissions.
-| Virt-who Reporter | Upload reports generated by virt-who to {Project}.
-It can be used if you configure virt-who manually and require a user role that has limited virt-who permissions.
-| Virt-who Viewer | View virt-who configurations.
-Users with this role can deploy virt-who instances using existing virt-who configurations.
+
+|Tasks manager
+|View and edit {Project} tasks.
```

Comment:
I'd either go with "tasks in {Project}" or just drop "{Project}" because it's kinda obvious.

Same for (all) other instances.

Reply:
I'll drop it, thanks.